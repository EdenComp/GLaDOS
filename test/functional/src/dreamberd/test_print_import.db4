print("import \"std:Format\";");
println("Bug");