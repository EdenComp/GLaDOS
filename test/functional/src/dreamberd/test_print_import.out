import "std:Format";Environment println does not exist
